src/visualizers/widgets/OperationCodeEditor/OperationCodeEditorWidget.js

```diff
@@ -20,13 +20,17 @@ define([
         // Add the shift-enter command
         this.editor.addCommand(
             monaco.KeyMod.Shift | monaco.KeyCode.Enter,
-            this.executeOrStopJob
+            this.handleOperationExecute.bind(this)
         );
         this.editor.updateOptions({
             lineNumbers: this.updateOffset.bind(this)
         });
     };
 
+    OperationCodeEditorWidget.prototype.handleOperationExecute = function () {
+        return this.executeOrStopJob(arguments);
```

Comment:
I believe this should be `...arguments` instead. The current approach will pass a single argument - the array-like arguments passed to `handleOperationExecute`.

Reply:
This has been added.